src/Rubric/RubCharacterBlockScanner.class.st

```diff
@@ -100,9 +100,9 @@ RubCharacterBlockScanner >> cr [
 				baselineY := line lineHeight.
 				lastCharacter := nil.
 				characterPoint := (nextLeftMargin ifNil: [leftMargin]) @ destY.
-				(lastIndex < text size and: [(text at: lastIndex) = Character cr and: [(text at: lastIndex+1) = Character lf]])
-					ifTrue: [lastIndex := lastIndex + 2]
-					ifFalse: [lastIndex := lastIndex + 1].
+				lastIndex := (lastIndex < text size and: [(text at: lastIndex) = Character cr and: [(text at: lastIndex+1) = Character lf]])
+					ifTrue: [ lastIndex + 2 ]
+					ifFalse: [ lastIndex + 1 ].
```

Comment:
you could as well factor `lastIndex := lastIndex + (... ifTrue: [2] ifFalse: [1]).`